.github/workflows/ci.yaml

```diff
@@ -90,4 +90,4 @@ jobs:
         uses: actions/upload-artifact@v1
         with:
           name: ci-artifacts
-          path: artifacts-jvm${{ matrix.java }}.zip
\ No newline at end of file
+          path: artifacts-code-start${{ matrix.java }}.zip
```

Comment:
The changes in `ci.yaml` are required to see the logs for each workflow action. Something we had in the windows PR, but we didn't commit.

Reply:
+1